src/Octoshift/ReclaimService.cs

```diff
@@ -162,9 +162,13 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                     continue;
                 }
 
-                var claimantId = await _githubApi.GetUserId(claimantLogin);
+                string claimantId;
```

Comment:
I think this is the best way to deal with the scoping of the `try` block - but I'm not a C# expert, so do feel free to give feedback on this.

Reply:
yeah this is pretty much the way